packages/orchestration/README.md

```diff
@@ -36,6 +43,15 @@ The orchestration service provides essential features like templating and conten
 
 Find more details about orchestration workflow [here](https://help.sap.com/docs/sap-ai-core/sap-ai-core-service-guide/orchestration-workflow).
 
+## Relationship between Orchestration and Resource Groups
+
+Access to orchestration of generative AI models is provided under the global AI scenario `orchestration`, which is managed by SAP AI Core.
+A deployment can be created to enable orchestration capabilities only with access to the global AI scenario `orchestration`.
+[Resource groups](https://help.sap.com/docs/sap-ai-core/sap-ai-core-service-guide/resource-groups?q=resource+group) represent a virtual collection of related resources within the scope of one SAP AI Core tenant.
+Each resource group allows for a one-time orchestration deployment.
+
+Consequently, each orchestration deployment uniquely maps to a resource group within the `orchestration` scenario.
```

Comment:
[pp] Maybe also link the other section where we tell about fetching using resource group

Reply:
As we haven't explained how to initialise the client yet, I wouldn't link the fetching section here.
I can instead link this in the fetching section.